Index: Client/src/components/adminComponents/EventsList.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/EventsList.jsx b/Client/src/components/adminComponents/EventsList.jsx
new file mode 100644
--- /dev/null	(date 1670527060276)
+++ b/Client/src/components/adminComponents/EventsList.jsx	(date 1670527060276)
@@ -0,0 +1,30 @@
+import React from 'react'
+import {
+    List,
+    Datagrid,
+    TextField,
+    EditButton,
+    DeleteButton,
+    EmailField,
+    DateField,
+} from 'react-admin';
+function EventsList(props) {
+    //console.log(props)
+    return (
+        <List {...props}>
+            <Datagrid>
+                <TextField source="eventID"/>
+                <TextField source="venueID"/>
+                <TextField source="title" />
+                <TextField source="price" />
+                <TextField source="description"/>
+                <TextField source="presenter"/>
+                <DateField source="dateTime"/>
+                <EditButton label="Edit" />
+                <DeleteButton label="Delete"/>
+            </Datagrid>
+        </List>
+    );
+}
+
+export default EventsList
\ No newline at end of file
Index: Client/src/components/adminComponents/EventsEdit.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/EventsEdit.jsx b/Client/src/components/adminComponents/EventsEdit.jsx
new file mode 100644
--- /dev/null	(date 1670527035846)
+++ b/Client/src/components/adminComponents/EventsEdit.jsx	(date 1670527035846)
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { Edit,PasswordInput, SimpleForm, TextInput,NumberInput, DateInput } from 'react-admin';
+
+function EventsEdit(props) {
+    return (
+        <Edit {...props}>
+            <SimpleForm>
+                <TextInput required source="title" />
+                <TextInput required source="presenter"/>
+                <TextInput required source="description"/>
+                <NumberInput required source="price" />
+                <DateInput required source="dateTime"/>
+
+            </SimpleForm>
+        </Edit>
+    );
+}
+
+export default EventsEdit
\ No newline at end of file
Index: Client/src/components/adminComponents/VenuesList.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/VenuesList.jsx b/Client/src/components/adminComponents/VenuesList.jsx
new file mode 100644
--- /dev/null	(date 1670527105713)
+++ b/Client/src/components/adminComponents/VenuesList.jsx	(date 1670527105713)
@@ -0,0 +1,27 @@
+import React from 'react'
+import {
+    List,
+    Datagrid,
+    TextField,
+    EditButton,
+    DeleteButton,
+    EmailField,
+    DateField,
+} from 'react-admin';
+function VenuesList(props) {
+    //console.log(props)
+    return (
+        <List {...props}>
+            <Datagrid>
+                <TextField source="venueID"/>
+                <TextField source="venueName" />
+                <TextField source="latitude" />
+                <TextField source="longitude"/>
+                <EditButton label="Edit" />
+                <DeleteButton label="Delete"/>
+            </Datagrid>
+        </List>
+    );
+}
+
+export default VenuesList
\ No newline at end of file
Index: Client/src/components/adminComponents/VenuesEdit.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/VenuesEdit.jsx b/Client/src/components/adminComponents/VenuesEdit.jsx
new file mode 100644
--- /dev/null	(date 1670527092773)
+++ b/Client/src/components/adminComponents/VenuesEdit.jsx	(date 1670527092773)
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { Edit,PasswordInput, SimpleForm, TextInput } from 'react-admin';
+
+function VenuesEdit(props) {
+    return (
+        <Edit {...props}>
+            <SimpleForm>
+                <TextInput source="venueName" />
+                <TextInput source="latitude" />
+                <TextInput source="longitude"/>
+            </SimpleForm>
+        </Edit>
+    );
+}
+
+export default VenuesEdit
\ No newline at end of file
Index: Client/src/components/adminComponents/createNewVenue.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/createNewVenue.jsx b/Client/src/components/adminComponents/createNewVenue.jsx
new file mode 100644
--- /dev/null	(date 1670527119643)
+++ b/Client/src/components/adminComponents/createNewVenue.jsx	(date 1670527119643)
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Create, EmailField, PasswordInput, SimpleForm, TextInput } from 'react-admin';
+
+function CreateNewVenue(props) {
+    return (
+        <Create {...props}>
+            <SimpleForm>
+                <TextInput source="venueName" />
+                <TextInput source="latitude" />
+                <TextInput source="longitude"/>
+            </SimpleForm>
+        </Create>
+    );
+}
+
+export default CreateNewVenue
\ No newline at end of file
Index: Client/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/.env b/Client/.env
new file mode 100644
--- /dev/null	(date 1670509277942)
+++ b/Client/.env	(date 1670509277942)
@@ -0,0 +1,1 @@
+REACT_APP_NEXT_PUBLIC_GOOGLE_MAPS_API_KEY="AIzaSyApdnJDwryPTna8oMfFP1om6WCmIWwASQ0"
\ No newline at end of file
Index: Server/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>ACCESS_TOKEN_SECRET=\"d6b4d96b72aa172fc94ed4b525b60f1930aff222e6fe40fb6d5cf7eb8c62efe12f42814f58f708acf537872f892e43e408ca9148186c227251bf26e9175ca70c\"\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Server/.env b/Server/.env
--- a/Server/.env	(revision cf977fd7349443a865182e6c549b28181f1937c2)
+++ b/Server/.env	(date 1670591681401)
@@ -1,1 +1,3 @@
 ACCESS_TOKEN_SECRET="d6b4d96b72aa172fc94ed4b525b60f1930aff222e6fe40fb6d5cf7eb8c62efe12f42814f58f708acf537872f892e43e408ca9148186c227251bf26e9175ca70c"
+MONGO_URI="mongodb+srv://Kaan:Kaan@cluster0.bw4l0fw.mongodb.net/?retryWrites=true&w=majority"
+PORT=4000
\ No newline at end of file
Index: Client/src/components/adminComponents/UsersEdit.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react'\r\n\r\nimport { Edit, PasswordInput, SimpleForm, TextInput } from 'react-admin';\r\n\r\nfunction UsersEdit(props) {\r\n    return (\r\n        <Edit {...props}>\r\n          <SimpleForm>\r\n            <TextInput disabled source=\"id\" />\r\n            <TextInput required source=\"username\" />\r\n            <PasswordInput required source=\"password\"/>\r\n          </SimpleForm>\r\n        </Edit>\r\n    );\r\n}\r\n\r\nexport default UsersEdit\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/UsersEdit.jsx b/Client/src/components/adminComponents/UsersEdit.jsx
--- a/Client/src/components/adminComponents/UsersEdit.jsx	(revision cf977fd7349443a865182e6c549b28181f1937c2)
+++ b/Client/src/components/adminComponents/UsersEdit.jsx	(date 1670527923241)
@@ -1,17 +1,17 @@
 import React from 'react'
 
-import { Edit, PasswordInput, SimpleForm, TextInput } from 'react-admin';
+import { Edit,PasswordInput, SimpleForm, TextInput } from 'react-admin';
 
 function UsersEdit(props) {
     return (
         <Edit {...props}>
-          <SimpleForm>
-            <TextInput disabled source="id" />
-            <TextInput required source="username" />
-            <PasswordInput required source="password"/>
-          </SimpleForm>
+            <SimpleForm>
+                <TextInput required source="username" />
+                <TextInput required source="email" />
+                <PasswordInput required source="password"/>
+            </SimpleForm>
         </Edit>
     );
 }
 
-export default UsersEdit
+export default UsersEdit
\ No newline at end of file
Index: Client/src/components/adminComponents/UsersList.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react'\r\nimport {\r\n    List,\r\n    Datagrid,\r\n    TextField,\r\n    EditButton,\r\n    DeleteButton,\r\n    EmailField,\r\n  } from 'react-admin';\r\nfunction UsersList(props) {\r\n    //console.log(props)\r\n    return (\r\n        <List {...props}>\r\n          <Datagrid>\r\n            <TextField source=\"id\" />\r\n            <TextField source=\"username\" />\r\n            <TextField source=\"name\" />\r\n            <EmailField source=\"email\"/>\r\n            <EditButton label=\"Edit\" />\r\n            <DeleteButton label=\"Delete\"/>\r\n          </Datagrid>\r\n        </List>\r\n      );\r\n}\r\n\r\nexport default UsersList\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/UsersList.jsx b/Client/src/components/adminComponents/UsersList.jsx
--- a/Client/src/components/adminComponents/UsersList.jsx	(revision cf977fd7349443a865182e6c549b28181f1937c2)
+++ b/Client/src/components/adminComponents/UsersList.jsx	(date 1670527077474)
@@ -6,21 +6,22 @@
     EditButton,
     DeleteButton,
     EmailField,
-  } from 'react-admin';
+} from 'react-admin';
 function UsersList(props) {
     //console.log(props)
     return (
         <List {...props}>
-          <Datagrid>
-            <TextField source="id" />
-            <TextField source="username" />
-            <TextField source="name" />
-            <EmailField source="email"/>
-            <EditButton label="Edit" />
-            <DeleteButton label="Delete"/>
-          </Datagrid>
+            <Datagrid>
+                <TextField source="id" />
+                <TextField source="username" />
+                <TextField source="name" />
+                <EmailField source="email"/>
+                <TextField source="password"/>
+                <EditButton label="Edit" />
+                <DeleteButton label="Delete"/>
+            </Datagrid>
         </List>
-      );
+    );
 }
 
-export default UsersList
+export default UsersList
\ No newline at end of file
Index: Client/src/components/adminComponents/AdminPage.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react'\r\nimport dataProvider from '../../dataProvider'\r\nimport {Admin, Resource} from 'react-admin'\r\nimport UsersList from './UsersList'\r\nimport UsersEdit from './UsersEdit'\r\nimport CreateNewUser from './CreateNewUser'\r\nfunction AdminPage() {\r\n  \r\n    return (\r\n      \r\n      <Admin basename=\"/admin-page\" dataProvider={dataProvider} >\r\n        <Resource\r\n                name=\"User List\"\r\n                list={UsersList}\r\n                edit={UsersEdit}\r\n                create={CreateNewUser}\r\n        />\r\n        <Resource\r\n                name=\"Event List\"\r\n        />\r\n      </Admin>\r\n    )\r\n}\r\n\r\nexport default AdminPage\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/AdminPage.jsx b/Client/src/components/adminComponents/AdminPage.jsx
--- a/Client/src/components/adminComponents/AdminPage.jsx	(revision cf977fd7349443a865182e6c549b28181f1937c2)
+++ b/Client/src/components/adminComponents/AdminPage.jsx	(date 1670528092505)
@@ -1,25 +1,41 @@
 import React from 'react'
-import dataProvider from '../../dataProvider'
+import userDataProvider from '../../dataProviders/adminDataProviderForUser'
 import {Admin, Resource} from 'react-admin'
 import UsersList from './UsersList'
+import EventsList from './EventsList'
+import VenuesList from './VenuesList'
 import UsersEdit from './UsersEdit'
+import EventsEdit from './EventsEdit'
+import VenuesEdit from './VenuesEdit'
 import CreateNewUser from './CreateNewUser'
+import CreateNewEvent from './CreateNewEvent'
+import CreateNewVenue from './createNewVenue'
 function AdminPage() {
-  
+
     return (
-      
-      <Admin basename="/admin-page" dataProvider={dataProvider} >
-        <Resource
+
+        <Admin basename="/admin-page" dataProvider={userDataProvider} >
+            <Resource
                 name="User List"
                 list={UsersList}
                 edit={UsersEdit}
                 create={CreateNewUser}
-        />
-        <Resource
+            />
+            <Resource
                 name="Event List"
-        />
-      </Admin>
+                list={EventsList}
+                edit={EventsEdit}
+                create={CreateNewEvent}
+
+            />
+            <Resource
+                name="Venue List"
+                list={VenuesList}
+                edit={VenuesEdit}
+                create={CreateNewVenue}
+            />
+        </Admin>
     )
 }
 
-export default AdminPage
+export default AdminPage
\ No newline at end of file
Index: Client/src/components/adminComponents/CreateNewUser.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react'\r\nimport { Create, EmailField, PasswordInput, SimpleForm, TextInput } from 'react-admin';\r\nfunction CreateNewUser(props) {\r\n    return (\r\n        <Create {...props}>\r\n          <SimpleForm>\r\n            <TextInput source=\"name\" required />\r\n            <TextInput source=\"username\" required />\r\n            <EmailField source=\"email\" required/>\r\n            <PasswordInput source=\"password\" required/>\r\n          </SimpleForm>\r\n        </Create>\r\n      );\r\n}\r\n\r\nexport default CreateNewUser\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/CreateNewUser.jsx b/Client/src/components/adminComponents/CreateNewUser.jsx
--- a/Client/src/components/adminComponents/CreateNewUser.jsx	(revision cf977fd7349443a865182e6c549b28181f1937c2)
+++ b/Client/src/components/adminComponents/CreateNewUser.jsx	(date 1670527910173)
@@ -3,14 +3,14 @@
 function CreateNewUser(props) {
     return (
         <Create {...props}>
-          <SimpleForm>
-            <TextInput source="name" required />
-            <TextInput source="username" required />
-            <EmailField source="email" required/>
-            <PasswordInput source="password" required/>
-          </SimpleForm>
+            <SimpleForm>
+                <TextInput source="name" required />
+                <TextInput source="username" required />
+                <TextInput source="email" required />
+                <PasswordInput source="password" required/>
+            </SimpleForm>
         </Create>
-      );
+    );
 }
 
-export default CreateNewUser
+export default CreateNewUser
\ No newline at end of file
Index: Client/src/components/adminComponents/CreateNewEvent.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/src/components/adminComponents/CreateNewEvent.jsx b/Client/src/components/adminComponents/CreateNewEvent.jsx
new file mode 100644
--- /dev/null	(date 1670526944576)
+++ b/Client/src/components/adminComponents/CreateNewEvent.jsx	(date 1670526944576)
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Create, DateInput, EmailField, NumberInput, PasswordInput, ReferenceInput, SelectInput, SimpleForm, TextInput } from 'react-admin';
+
+function CreateNewEvent(props) {
+
+    return (
+        <Create {...props}>
+
+            <SimpleForm>
+                <TextInput required source="title"  />
+                <TextInput required source="presenter" />
+                <NumberInput required source="venueID" />
+                <TextInput required source='description'/>
+                <NumberInput required source='price' />
+                <DateInput required source="dateTime" />
+            </SimpleForm>
+        </Create>
+    );
+}
+
+export default CreateNewEvent
\ No newline at end of file
