Index: Client/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/.env b/Client/.env
new file mode 100644
--- /dev/null	(date 1670617815656)
+++ b/Client/.env	(date 1670617815656)
@@ -0,0 +1,1 @@
+REACT_APP_NEXT_PUBLIC_GOOGLE_MAPS_API_KEY="AIzaSyApdnJDwryPTna8oMfFP1om6WCmIWwASQ0"
\ No newline at end of file
